gunicorn/http/message.py

```diff
@@ -2,19 +2,19 @@
 #
 # This file is part of gunicorn released under the MIT license.
 # See the NOTICE for more information.
-
-import io
+import ipaddress
 import re
 import socket
+import struct
 
 from gunicorn.http.body import ChunkedReader, LengthReader, EOFReader, Body
 from gunicorn.http.errors import (
     InvalidHeader, InvalidHeaderName, NoMoreData,
     InvalidRequestLine, InvalidRequestMethod, InvalidHTTPVersion,
     LimitRequestLine, LimitRequestHeaders,
 )
-from gunicorn.http.errors import InvalidProxyLine, ForbiddenProxyRequest
-from gunicorn.http.errors import InvalidSchemeHeaders
+from gunicorn.http.errors import InvalidProxyHeader, InvalidProxyLine
```

Comment:
why this change?

Reply:
Hi Benoit, looking over this twice I am not sure which change you mean or what the problem with it is? Do you mean why I introduced `InvalidProxyHeader` or why I changed the order?

Reply:
i was thinking to the import removed, but I need to re-read the code. I guess they were not used anymore?